WalletWasabi/TorSocks5/TorSocks5Client.cs

```diff
@@ -113,13 +113,13 @@ internal async Task ConnectAsync()
 		/// </summary>
 		internal async Task HandshakeAsync(bool isolateStream = true)
 		{
-			if (!isolateStream)
+			if (isolateStream)
 			{
-				await HandshakeAsync("");
+				await HandshakeAsync(RandomString.Generate(21));
 			}
 			else
 			{
-				await HandshakeAsync(RandomString.Generate(21));
+				await HandshakeAsync("");
```

Comment:
This could function could be changed to just `await HandshakeAsync(isolateStream ? RandomString.Generate(21) : "")`

Reply:
Good catch. ACK.

Reply:
Could be in separate PR though, not 100 percent sure this'll get in yet.

Reply:
ACK for this file btw.

Reply:
I edited it.